google-cloud-pubsub/samples/acceptance/topics_test.rb

```diff
@@ -35,6 +43,126 @@
+                                                   resource_group: "fake-resource-group",
+                                                   namespace: "fake-namespace",
+                                                   event_hub: "fake-event-hub",
+                                                   client_id: "11111111-1111-1111-1111-11111111111",
+                                                   tenant_id: "22222222-2222-2222-2222-222222222222",
+                                                   subscription_id: "33333333-3333-3333-3333-333333333333",
+                                                   gcp_service_account: gcp_service_account
+    end
+  end
+
+  it "supports pubsub_create_topic_with_confluent_cloud_ingestion" do
+    # pubsub_create_topic_with_confluent_cloud_ingestion
+    assert_output "Topic with Confluent Cloud Ingestion projects/#{pubsub.project}/topics/#{topic_id} created.\n" do
+      create_topic_with_confluent_cloud_ingestion topic_id: topic_id,
+                                                  bootstrap_server: "fake-bootstrap-server-id.us-south1.gcp.confluent.cloud:9092",
+                                                  cluster_id: "fake-cluster-id",
+                                                  confluent_topic: "fake-confluent-topic-name",
+                                                  identity_pool_id: "fake-identity-pool-id",
+                                                  gcp_service_account: gcp_service_account
+    end
+  end
+
+  it "supports pubsub_create_topic, pubsub_update_topic_type" do
+    # pubsub_create_topic
+    assert_output "Topic projects/#{pubsub.project}/topics/#{topic_id} created.\n" do
+      create_topic topic_id: topic_id
+    end
+
+    # pubsub_update_topic_type
+    assert_output "Topic projects/#{pubsub.project}/topics/#{topic_id} updated.\n" do
+      update_topic_type topic_id: topic_id,
+                        stream_arn: "arn:aws:kinesis:us-west-2:111111111111:stream/fake-stream-name",
+                        consumer_arn: "arn:aws:kinesis:us-west-2:111111111111:stream/fake-stream-name/consumer/consumer-1:1111111111",
+                        aws_role_arn: aws_role_arn,
+                        gcp_service_account: gcp_service_account
+    end
+  end
+
+  it "supports pubsub_create_topic, pubsub_create_cloud_storage_subscription" do
+    # pubsub_create_topic
+    assert_output "Topic projects/#{pubsub.project}/topics/#{topic_id} created.\n" do
+      create_topic topic_id: topic_id
+    end
+
+    subscription_id = random_subscription_id
+    bucket_id = random_bucket_id
+
+    # pubsub_create_cloud_storage_subscription
+    assert_output "Cloud storage subscription #{subscription_id} created.\n" do
+      create_cloud_storage_subscription topic_id: topic_id, subscription_id: subscription_id, bucket: bucket_id 
+    end
+    @subscription = subscription_admin.get_subscription subscription: pubsub.subscription_path(subscription_id)
```

Comment:
Similarly, you can use `subscription` (a local variable) here instead of `@subscription` (an instance variable).